### DISCUSSION TITLE
how to create a table view that is sourced from another connection

### DISCUSSION THREAD

**Discussion by romakelap** [dbeaver/dbeaver#21756] [Q&A]
currently I am in the database1 connection and I want to create a table view that is sourced from another connection. How do I create it in dbeaver

CREATE VIEW csvConnection AS
select * from ???

and this is another connection name = csvConnection

**Comment by LonwoLonwo** [comment]
Hello @romakelap 

You can not do it in DBeaver. Your view must have a source in the connection of the view.
